pkg/projects/util.go

```diff
@@ -171,9 +171,17 @@ func Int64Ptr(v int64) *int64 { return &v }
 func StringPtr(v string) *string { return &v }
 
 func BytesToGigabytles(bytes int) int {
- return int(bytes/int(math.Pow(1024, 3)))
+	if bytes == -1 {
```

Comment:
This should actually be any value less than `bytes <= -1` Anything `< 0` is invalid

Reply:
Not according to the API. Unsetting this field in the UI returns -1 in the API 🤦

Reply:
Yeah, but your code can easily catch the impossible scenario - I think you should just do it because it handles the entire number domain